tests/test_checks.py

```diff
@@ -117,3 +124,13 @@ def test_season(self, tasmin_series):
 
         miss = checks.missing_any(ts, freq='YS', season='SON')
         np.testing.assert_equal(miss, [False])
+
+    def test_hydro(self):
+        fn = os.path.join(TESTS_DATA, 'Raven', 'q_sim.nc')
+        ds = xr.open_dataset(fn)
+        miss = checks.missing_any(ds.q_sim, freq='YS')
+        np.testing.assert_array_equal(miss[:-1], False)
```

Comment:
since you did `miss[:-1]`, you extract the last value of the array, so it's not an array anymore, should this be just `assert_equal`?